### PR TITLE
Moar sites

### PR DESCRIPTION
**Pull request by bryanbell**

### CODE REVIEW

**Comment by pdehaan** [comment]
```
  1) top_sites #27 ESPN should have an image_url pointing to a .png or .svg image prefixed with images/:
     AssertionError: expected espn-go-com.png to be found on disk: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)

  2) top_sites #120 FiveThirtyEight should have background color as a hex color:
     AssertionError: expected black to be a hex color: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:36:16)

  3) top_sites #134 JCPenney should have an image_url pointing to a .png or .svg image prefixed with images/:
     AssertionError: expected jcpenney-com.png to be found on disk: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
```

File: top_sites.json [hunk 1]
```diff
@@ -1,567 +1,1206 @@
...
+    "url": "http://faithtap.com/",
+    "image_url": "faithtap-com.png",
+    "background_color": "#4c286f",
+    "domain": "faithtap.com"
+  },
+  {
+    "title": "FiveThirtyEight",
+    "url": "http://fivethirtyeight.com/",
+    "image_url": "fivethirtyeight-com.png",
+    "background_color": "black",
```
Comment (pdehaan) [inline top_sites.json]: Needs to be "#000"

File: top_sites.json [hunk 2]
```diff
@@ -1,567 +1,1206 @@
...
     "url": "http://www.ebay.com",
     "image_url": "ebay-com.png",
-    "background_color": "#ededed"
+    "background_color": "#ededed",
+    "domain": "ebay.com"
   },
   {
     "title": "ESPN",
     "url": "http://espn.go.com",
     "image_url": "espn-go-com.png",
```
Comment (pdehaan) [inline top_sites.json:187]: I think should be "espn.go-com.png"

File: top_sites.json [hunk 3]
```diff
@@ -1,567 +1,1206 @@
...
+    "title": "IKEA",
+    "url": "http://www.ikea.com/",
+    "image_url": "ikea-com.png",
+    "background_color": "#00329c",
+    "domain": "ikea.com"
+  },
+  {
+    "title": "JCPenney",
+    "url": "http://www.jcpenney.com/",
+    "image_url": "jcpenney-com.png",
```
Comment (pdehaan) [inline top_sites.json:955]: Image is named "jcpenny-com.png" on disk so we'll need to rename the image to add the missing "e".

**Comment by pdehaan** [comment]
Looks good. A few minor issues with the data, and needs a rebase, but I can take a stab at that after my ~~death march~~ meeting is finished.

**Comment by pdehaan** [comment]
I think my only other question is why we need the `domain` field, versus can we remove that from the JSON and inject it in the index.js script when a user requires the module (same as how we break out the RGB values into an array).

**Comment by pdehaan** [comment]
~~Not sure if we want to run the images through a minifier. It looks like the average filesize is 16KB (and a total filesize of 2.7MB for images/*):~~

**UPDATE:** Disregard that, it looks like the images were always that size. Not sure why I was thinking they were ~5KB each, so this isn't blocking the PR in any way.

| TITLE | IMAGE_URL | SIZE | EXT |
| --- | --- | --: | :-: |
| About | images/about-com.png | 16.9 KB | .png |
| Adobe | images/adobe-com.png | 15.2 KB | .png |
| Amazon | images/amazon-com.png | 18.3 KB | .png |
| Amazon Web Services | images/amazonaws-com.png | 16.4 KB | .png |
| American Express | images/americanexpress-com.png | 18.6 KB | .png |
| AOL | images/aol-com.png | 17.6 KB | .png |
| Apple | images/apple-com.png | 15.2 KB | .png |
| Ask.com | images/ask-com.png | 17.1 KB | .png |
| AT&T | images/att-com.png | 22.3 KB | .png |
| Bank of America | images/bankofamerica-com.png | 15.4 KB | .png |
| Best Buy | images/bestbuy-com.png | 18.1 KB | .png |
| Bing | images/bing-com.png | 15.2 KB | .png |
| Blogger | images/blogger-com.png | 15.1 KB | .png |
| Business Insider | images/businessinsider-com.png | 15.5 KB | .png |
| BuzzFeed | images/buzzfeed-com.png | 15.7 KB | .png |
| Buzzlie | images/buzzlie-com.png | 16.9 KB | .png |
| Capital One | images/capitalone-com.png | 17.5 KB | .png |
| Chase | images/chase-com.png | 15 KB | .png |
| Citibank | images/citi-com.png | 17.6 KB | .png |
| CNET | images/cnet-com.png | 16.3 KB | .png |
| CNN | images/cnn-com.png | 15.7 KB | .png |
| Craigslist | images/craigslist-org.png | 16 KB | .png |
| Diply.com | images/diply-com.png | 17.8 KB | .png |
| Discover | images/discovercard-com.png | 16.7 KB | .png |
| Dropbox | images/dropbox-com.png | 15.6 KB | .png |
| eBay | images/ebay-com.png | 14.6 KB | .png |
| ESPN | images/espn-go-com.png | 14.6 KB | .png |
| Etsy | images/etsy-com.png | 15.1 KB | .png |
| Facebook | images/facebook-com.png | 14.8 KB | .png |
| Fandom | images/wikia-com.png | 17.2 KB | .png |
| FedEx | images/fedex-com.png | 15 KB | .png |
| Forbes | images/forbes-com.png | 15 KB | .png |
| Fox News | images/foxnews-com.png | 16.7 KB | .png |
| Gfycat | images/gfycat-com.png | 15.6 KB | .png |
| GitHub | images/github-com.png | 15.4 KB | .png |
| Go | images/go-com.png | 15.6 KB | .png |
| Google | images/google-com.png | 17.2 KB | .png |
| Google Images | images/images-google-com.png | 17.2 KB | .png |
| Groupon | images/groupon-com.png | 15.4 KB | .png |
| Hacker News | images/news-ycombinator-com.png | 14.9 KB | .png |
| Home Depot | images/homedepot-com.png | 18.9 KB | .png |
| The Huffington Post | images/huffingtonpost-com.png | 17.6 KB | .png |
| Hulu | images/hulu-com.png | 14.5 KB | .png |
| IMDb | images/imdb-com.png | 14.8 KB | .png |
| Imgur | images/imgur-com.png | 15.8 KB | .png |
| Instagram | images/instagram-com.png | 15.1 KB | .png |
| Instructure | images/instructure-com.png | 14.3 KB | .png |
| Internal Revenue Service | images/irs-gov.png | 17.1 KB | .png |
| Intuit | images/intuit-com.png | 14.9 KB | .png |
| Kohl's | images/kohls-com.png | 14.9 KB | .png |
| LinkedIn | images/linkedin-com.png | 14.8 KB | .png |
| Live | images/live-com.png | 15.3 KB | .png |
| Lowes | images/lowes-com.png | 15 KB | .png |
| Macy's | images/macys-com.png | 15.1 KB | .png |
| Mail Online | images/dailymail-co-uk.png | 15.1 KB | .png |
| Microsoft | images/microsoft-com.png | 14.5 KB | .png |
| Microsoft Office | images/office-com.png | 14.9 KB | .png |
| Microsoft Online | images/microsoftonline-com.png | 14.9 KB | .png |
| MSN.com | images/msn-com.png | 15.4 KB | .png |
| National Institutes of Health (NIH) | images/nih-gov.png | 15.4 KB | .png |
| Netflix | images/netflix-com.png | 14.7 KB | .png |
| The New York Times | images/nytimes-com.png | 15.7 KB | .png |
| Pandora | images/pandora-com.png | 15.1 KB | .png |
| Patch | images/patch-com.png | 15.6 KB | .png |
| PayPal | images/paypal-com.png | 15.4 KB | .png |
| Pinterest | images/pinterest-com.png | 15.5 KB | .png |
| Realtor | images/realtor-com.png | 17.2 KB | .png |
| Reddit | images/reddit-com.png | 17.8 KB | .png |
| Salesforce.com | images/salesforce-com.png | 15.7 KB | .png |
| Slickdeals | images/slickdeals-net.png | 16 KB | .png |
| Stack Overflow | images/stackoverflow-com.png | 15.3 KB | .png |
| Target | images/target-com.png | 15.6 KB | .png |
| TripAdvisor | images/tripadvisor-com.png | 17 KB | .png |
| Tumblr | images/tumblr-com.png | 14.7 KB | .png |
| Twitch | images/twitch-tv.png | 14.8 KB | .png |
| Twitter | images/twitter-com.png | 15.3 KB | .png |
| UPS | images/ups-com.png | 15.9 KB | .png |
| USA TODAY | images/usatoday-com.png | 15 KB | .png |
| USPS | images/usps-com.png | 15.4 KB | .png |
| Verizon Wireless | images/verizonwireless-com.png | 15.1 KB | .png |
| VICE | images/vice-com.png | 15.2 KB | .png |
| Walmart.com | images/walmart-com.png | 15.4 KB | .png |
| Washington Post | images/washingtonpost-com.png | 15.6 KB | .png |
| The Weather Channel | images/weather-com.png | 15.9 KB | .png |
| WebMD | images/webmd-com.png | 16.7 KB | .png |
| Wells Fargo | images/wellsfargo-com.png | 15.1 KB | .png |
| Wikipedia | images/wikipedia-org.png | 15.3 KB | .png |
| WittyFeed | images/wittyfeed-com.png | 16 KB | .png |
| WordPress.com | images/wordpress-com.png | 16.4 KB | .png |
| Xfinity | images/xfinity-com.png | 15.3 KB | .png |
| Xfinity | images/xfinity-com.png | 15.3 KB | .png |
| Yahoo | images/yahoo-com.png | 15.2 KB | .png |
| Yelp | images/yelp-com.png | 15.4 KB | .png |
| YouTube | images/youtube-com.png | 14.9 KB | .png |
| Zillow | images/zillow-com.png | 15.5 KB | .png |
| ABC News | images/abcnews-go-com.png | 18.6 KB | .png |
| AccuWeather | images/accuweather-com.png | 16 KB | .png |
| ADP.com | images/adp-com.png | 15.5 KB | .png |
| Allrecipes | images/allrecipes-com.png | 17 KB | .png |
| Airbnb | images/airbnb-com.png | 16.2 KB | .png |
| American Airlines | images/aa-com.png | 16.8 KB | .png |
| Ancestry | images/ancestry-com.png | 15.9 KB | .png |
| Answers.com | images/answers-com.png | 15.7 KB | .png |
| Baidu | images/baidu-com.png | 15.7 KB | .png |
| BBC | images/bbc-com.png | 14.9 KB | .png |
| Blackboard | images/blackboard-com.png | 15.5 KB | .png |
| Bleacher Report | images/bleacherreport-com.png | 15.1 KB | .png |
| Box | images/box-com.png | 15.3 KB | .png |
| California Home Page | images/ca-gov.png | 19.3 KB | .png |
| CBS News | images/cbsnews-com.png | 16.4 KB | .png |
| CBSSports.com | images/cbssports-com.png | 17.6 KB | .png |
| Conservative Tribune | images/conservativetribune-com.png | 17.2 KB | .png |
| Costco | images/costco-com.png | 15.5 KB | .png |
| Delta Air Lines | images/delta-com.png | 15.2 KB | .png |
| DeviantArt | images/deviantart-com.png | 14.8 KB | .png |
| Drudge Report | images/drudgereport-com.png | 17 KB | .png |
| Discover | images/discovercard-com.png | 16.7 KB | .png |
| Ebates | images/ebates-com.png | 17 KB | .png |
| Eventbrite | images/eventbrite-com.png | 15.1 KB | .png |
| Expedia | images/expedia-com.png | 15.6 KB | .png |
| FaithTap | images/faithtap-com.png | 15.7 KB | .png |
| FiveThirtyEight | images/fivethirtyeight-com.png | 15.8 KB | .png |
| Flickr | images/flickr-com.png | 15 KB | .png |
| Feedly | images/feedly-com.png | 15 KB | .png |
| Fitbit | images/fitbit-com.png | 15.5 KB | .png |
| Food Network | images/foodnetwork-com.png | 16.3 KB | .png |
| Gap | images/gap-com.png | 14.9 KB | .png |
| Gawker | images/gawker-com.png | 15.3 KB | .png |
| Goodreads | images/goodreads-com.png | 15.5 KB | .png |
| The Guardian | images/theguardian-com.png | 15.5 KB | .png |
| Gizmodo | images/gizmodo-com.png | 15.5 KB | .png |
| Glassdoor | images/glassdoor-com.png | 14.8 KB | .png |
| Houzz | images/houzz-com.png | 15.1 KB | .png |
| IGN | images/ign-com.png | 16.5 KB | .png |
| IKEA | images/ikea-com.png | 15.1 KB | .png |
| JCPenney | images/jcpenney-com.png | 15.2 KB | .png |
| Kayak | images/kayak-com.png | 14.9 KB | .png |
| Lifehacker | images/lifehacker-com.png | 14.6 KB | .png |
| Los Angeles Times | images/latimes-com.png | 15.4 KB | .png |
| NBC News | images/nbcnews-com.png | 16.7 KB | .png |
| MapQuest | images/mapquest-com.png | 15.1 KB | .png |
| Mashable | images/mashable-com.png | 14.8 KB | .png |
| MLB.com | images/mlb-com.png | 15 KB | .png |
| Newegg | images/newegg-com.png | 17.7 KB | .png |
| New York Post | images/nypost-com.png | 20.7 KB | .png |
| Nordstrom | images/nordstrom-com.png | 15.1 KB | .png |
| NPR | images/npr-org.png | 14.9 KB | .png |
| Overstock.com | images/overstock-com.png | 18.3 KB | .png |
| People.com | images/people-com.png | 15.9 KB | .png |
| POLITICO | images/politico-com.png | 14.7 KB | .png |
| Quora | images/quora-com.png | 15.6 KB | .png |
| Sears | images/sears-com.png | 15.6 KB | .png |
| Slate Magazine | images/slate-com.png | 15.5 KB | .png |
| SoundCloud | images/soundcloud-com.png | 15.3 KB | .png |
| Southwest | images/southwest-com.png | 17.4 KB | .png |
| Spotify | images/spotify-com.png | 15.8 KB | .png |
| Stack Exchange | images/stackexchange-com.png | 16.8 KB | .png |
| Staples | images/staples-com.png | 20.9 KB | .png |
| SurveyMonkey | images/surveymonkey-com.png | 15.2 KB | .png |
| Swagbucks | images/swagbucks-com.png | 15.7 KB | .png |
| Taboola | images/taboola-com.png | 15.5 KB | .png |
| Ticketmaster | images/ticketmaster-com.png | 17.4 KB | .png |
| T-Mobile | images/t-mobile-com.png | 14.9 KB | .png |
| Trulia | images/trulia-com.png | 15.1 KB | .png |
| Thesaurus.com | images/thesaurus-com.png | 17.4 KB | .png |
| USAA | images/usaa-com.png | 16.4 KB | .png |
| U.S. Bank | images/usbank-com.png | 15.3 KB | .png |
| Verizon | images/verizon-com.png | 15.2 KB | .png |
| Vimeo | images/vimeo-com.png | 15.4 KB | .png |
| The Wall Street Journal | images/wsj-com.png | 15.6 KB | .png |
| Wayfair | images/wayfair-com.png | 18.8 KB | .png |
| wikiHow | images/wikihow-com.png | 16.1 KB | .png |
| Wunderground | images/wunderground-com.png | 17.9 KB | .png |

TOTAL SIZE: 2.69 MB

**Comment by k88hudson** [comment]
Hm, 2.69MB is pretty big

**Comment by bryanbell** [comment]
I forgot to compress the files.  I'll do that soon.

**Comment by pdehaan** [comment]
Image optimization issue filed earlier as https://github.com/nchapman/tippy-top-sites/issues/20.
We can either check in optimized images or maybe run optimization on a npm/git prepublish hook.

**Pull request merged**